sa/sa_test.go

```diff
@@ -2061,13 +2056,12 @@ func TestRevokeCertificate(t *testing.T) {
 
 	fc.Add(1 * time.Hour)
 
-	now := fc.Now()
```

Comment:
I think this is a place where you took removing time-based variables a step too far. The point of my comments in the previous round was to remove places where we were declaring `now := fc.Now()`, and then _only using that value one_.

Here, and several other places in this file, we've now removed variables like `now` and `expires` which are used in _multiple_ places in the same test. For example, here in this particular test case, the value of `now` was used in three ways:
- to populate the `Date` field of the test input
- to check the `RevokedDate` of the test output
- to check the `OcspLastUpdated` of the test output

We want all three of those to be truly the same. If time passed during the test (e.g. because we can some concurrency issue and a different parallel test managed to increment the fake clock) this test would suddenly start failing. So storing a timestamp in a variable, and then using that same variable in all three places, is what we actually want to do.

This PR should only be removing these variables when they're only used in one spot, like up at the top of the PR in ca.go, not when they're appropriately re-used in multiple places.